app/assets/javascripts/quby/answers/validation.js

```diff
@@ -189,6 +208,48 @@
     return !failed;
   };
 
+  function parsePartialDate(inputs) {
+    var value = function(placeholder) {
+      var val = inputs.filter("[placeholder=" + placeholder + "]").first().val();
+      if(val === undefined || val == "")
```

Comment:
aan de andere kant als niets is ingevuld en hij is niet verplicht, dan is hij wel terecht geldig.